### PR TITLE
avatar path

### PR DESCRIPTION
**Pull request by xxyzz**
Support new probe API.

### CODE REVIEW

**Review by Anduin2017 — APPROVED** [review]

File: src/app/Models/ContactInfo.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 export class ContactInfo {
     public displayName: string;
-    public displayImageKey: number;
+    public displayImagePath: number;
```
Comment (Anduin2017) [inline src/app/Models/ContactInfo.ts]: Why is this a number?

File: src/app/Models/GroupsResults.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 export class GroupsResult {
-    public imageKey: number;
+    public imagePath: number;
```
Comment (Anduin2017) [inline src/app/Models/GroupsResults.ts]: Why is this a number?

File: src/app/Models/UploadFile.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { AiurProtocal } from './AiurProtocal';
 
 export class UploadFile extends AiurProtocal {
```
Comment (Anduin2017) [inline src/app/Models/UploadFile.ts:3]: It seems that this class does not exist anymore.

**Review by Anduin2017 — CHANGES_REQUESTED** [review]
It seems that not everything looks fine.

File: src/app/Models/UploadFile.ts (hunk 3 above)
Comment (xxyzz) [reply src/app/Models/UploadFile.ts:3]: ?

**Review by Anduin2017 — APPROVED** [review]

**Pull request merged**